src/hooks/useENSRegistrationCosts.ts

```diff
@@ -485,27 +487,21 @@ export default function useENSRegistrationCosts({
     yearsDuration,
   ]);
 
-  const gasFeeReady = useMemo(
-    () =>
-      !isEmpty(useGasGasFeeParamsBySpeed) && !isEmpty(useGasCurrentBlockParams),
-    [useGasCurrentBlockParams, useGasGasFeeParamsBySpeed]
-  );
-
   const { isSuccess, isLoading, isIdle } = useMemo(() => {
     const statusQueries = queries.slice(0, 2);
     const isSuccess =
       !statusQueries.some(a => a.status !== 'success') &&
       !!data?.estimatedRentPrice &&
-      gasFeeReady;
+      isGasReady;
```

Comment:
use of `isGasReady` from `useGas` which is fixing some loading issues described in the PR description